tests/v2/08_full_workflow/test_full_workflow_local_v2.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from common.common import failing_workflow_UnknownError
```

Comment:
Why a nested `common.common` structure? Could we simply use `common.py`?
(maybe with a more clear name like `common_functions`)